email/spool.rst

```diff
@@ -141,3 +141,16 @@ You can also set the time limit in seconds:
 Of course you will not want to run this manually in reality. Instead, the
 console command should be triggered by a cron job or scheduled task and run
 at a regular interval.
+
+.. caution::
+
+    When you use the filesystem for spooling, messages are serialized with
+    the message class name. If the service is lazy loaded this class name
+    change on every cache clear. So if you send a mail, then you clear the
+    cache, the message will not be unserialisable.
+
+    On the next ``swiftmailer:spool:send`` an error will raise because the
+    class ``Swift_Message_<someRandomCharacters>`` doesn't exist (anymore).
+
+    The solutions are either to use the memory spool or to load the
+    swiftmailer without ``lazy`` option.
```

Comment:
Something looks missing here:

```
... to load the swiftmailer without ...
```

"the siwftmailer" ... library? service?

And another question: about the `lazy` option. It's not explained in http://symfony.com/doc/current/reference/configuration/swiftmailer.html, so this may confuse some readers.

Reply:
Still a valid comment, I'm trying to figure out here which service I should change to adjust the `lazy` option.

Reply:
(as using the container debugger, there seems not to be a `swiftmailer` service registered)